src/pybind/mgr/dashboard/controllers/mgr_modules.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+from . import ApiController, RESTController, Endpoint, UpdatePermission
+from .. import mgr
+from ..security import Scope
+from ..services.ceph_service import CephService
+from ..services.exception import handle_send_command_error
+from ..tools import str_to_bool
+
+
+@ApiController('/mgr/module', Scope.CONFIG_OPT)
+class MgrModules(RESTController):
+    managed_modules = ['telemetry']
+
+    @handle_send_command_error('mgr_modules')
+    def list(self):
+        result = []
+        modules = CephService.send_command('mon', 'mgr module ls')
```

Comment:
Is it not possible to obtain this info from MgrMap?

Reply:
Fixed. Thanks for the hint.